account_statement_base/tests/test_account_statement_base.py

```diff
@@ -0,0 +1,28 @@
+from odoo.tests import tagged
+
+from odoo.addons.account.tests.common import TestAccountReconciliationCommon
+
+
+@tagged("post_install", "-at_install")
+class TestReconciliationWidget(TestAccountReconciliationCommon):
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.acc_bank_stmt_model = cls.env["account.bank.statement"]
```

Comment:
I would add a test in which you create a bank statement and then ensure you get it's move lines on the action's domain